src/mon/NVMeofGwMap.h

```diff
@@ -25,6 +25,12 @@
 #include "msg/Message.h"
 #include "common/ceph_time.h"
 #include "NVMeofGwTypes.h"
+#define dout_context g_ceph_context
+#define dout_subsys ceph_subsys_mon
+#undef dout_prefix
+#define MODULE_PREFFIX "nvmeofgw "
+#define dout_prefix *_dout << MODULE_PREFFIX << __PRETTY_FUNCTION__ << " "
```

Comment:
We don't really want to be #define'ing a bunch of dout related macros in a header.  If you feel you need the debug statements in encode/decode, move them into the .cc file.

Reply:
ok, agree . done